### PR TITLE
dev mode update 

### PR DESCRIPTION
**Pull request by DannyS03**

### CODE REVIEW

**Comment by NimG98** [comment]
Need to address: https://github.com/OpenLiberty/guide-cloud-aws/issues/77

File: README.adoc [hunk 1]
```diff
@@ -272,27 +268,35 @@ build log for any potential errors.
...
-docker login -u AWS -p [password_string] https://[aws_account_id].dkr.ecr.us-east-2.amazonaws.com
+aws sts get-caller-identity --output text --query 'Account'
 ```
 
-Run the `docker login` command that is returned to finish authenticating your Docker client.
 The `[aws_account_id]` is a unique 12-digit ID that is assigned to every AWS account. You 
 will notice this ID in the output from various commands because AWS uses it to differentiate 
 your resources from other accounts. 
 
+Replace the `[password_string]` and the `[aws_account_id]` the following
```
Comment (NimG98) [inline README.adoc]: put "in" before "the following"

File: finish/inventory/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java [hunk 2]
```diff
@@ -12,46 +12,45 @@
...
-
-public class InventoryEndpointTest {
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+@TestMethodOrder(OrderAnnotation.class)
+public class InventoryEndpointIT {
```
Comment (NimG98) [inline finish/inventory/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java]: put new line space between imports and class name

File: README.adoc (hunk 1 above)
Comment (DannyS03) [reply README.adoc]: updated.

File: finish/inventory/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java (hunk 2 above)
Comment (DannyS03) [reply finish/inventory/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java]: updated.

File: finish/inventory/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java [hunk 3]
```diff
@@ -12,28 +12,35 @@
...
 @RequestScoped
 public class SystemClient {
 
   // Constants for building URI to the system service.
-  private final int DEFAULT_PORT = Integer.valueOf(System.getProperty("default.http.port"));
   private final String SYSTEM_PROPERTIES = "/system/properties";
   private final String PROTOCOL = "http";
 
+  @Inject
+  @ConfigProperty(name = "system.http.port")
```
Comment (NimG98) [inline finish/inventory/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java]: needs to be default not system

Comment (NimG98) [reply finish/inventory/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java]: same with one in start

Comment (DannyS03) [reply finish/inventory/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java]: updated.

File: finish/system/src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java [hunk 4]
```diff
@@ -56,59 +56,39 @@ public boolean verify(String hostname, SSLSession session) {
...
         String greeting = response.getHeaderString("X-Pod-Name");
         
-        assertNotNull(
-            "Container name should not be null but it was. The service is probably not running inside a container",
-            greeting);
+        assertNotNull(greeting,
+            "Container name should not be null but it was. The service is probably not running inside a container");
+    }
+
+    private void assertResponse(String clusterUrl2, Response response2) {
```
Comment (NimG98) [inline finish/system/src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java]: - [ ] `assertResponse` function is blank
- [ ] `getResponse` function is returning null


these both should be like [this](https://github.com/OpenLiberty/guide-istio-intro/blob/master/finish/src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java#L116)

Comment (DannyS03) [reply finish/system/src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java]: addressed.

**Comment by DannyS03** [comment]
looks good on `qa`.

**Pull request merged**